manifests/broker/config.pp

```diff
@@ -24,7 +24,6 @@
     $config_notify = undef
   }
 
-  $doctag = 'brokerconfigs'
```

Comment:
Is that a dead variable that isn't used anywhere?

Reply:
Ahhh, that one I messed up I think. It is in the properties.erb but only to set the documentation link. So maybe we should keep that in?

Reply:
Yeah I think we should keep it